connectivity/check/logging.go

```diff
@@ -128,6 +128,14 @@ func (ct *ConnectivityTest) Fatalf(format string, a ...interface{}) {
 // should always call the methods implemented on Test.
 //
 
+// Progress adds a progress indicator if logging is buffered
+func (t *Test) Progress() {
```

Comment:
This should probably be unexported, we're not exposing it as API to the test suite.

Reply:
Unexported.